### DISCUSSION TITLE
Teams log in (change MS account)

### DISCUSSION THREAD

**Discussion by steveshallcross** [isaacrlevin/presencelight#825] [Q&A] [answered]
How do I change the MS account I'm using for Teams?
I need to switch from an old work account to my personal MS account.

I can't see anything in the settings.json file.

I just get a spinner in the 'Team Status' section in the App.

Version: 5.6.11
Installed from the App Store.

![image](https://github.com/isaacrlevin/presencelight/assets/55241320/7d140874-6e1b-498f-89de-36ea4bec7c8d)

**Comment by isaacrlevin** [comment]
Not 100% sure if personal Teams will work (the AAD tenant config is different). You will need to log out of PL (top right) and when you login again, login with your personal account.

**Reply by steveshallcross** [reply]
Thanks Isaac.
I got this when trying to sign in with a personal account, so it looks like it doesn't work.
![image](https://github.com/isaacrlevin/presencelight/assets/55241320/d3204ba4-d8a0-4026-b7ee-776b9e740506)

**Reply by steveshallcross** [reply]
However, there is no "top right log out" option that I can see.
![image](https://github.com/isaacrlevin/presencelight/assets/55241320/32256795-b83c-4c51-a523-8b64c55ee8db)

**Comment by isaacrlevin** [comment]
Look in this directory for a "msalcache.bin" file

C:\Users\{USER}\AppData\Local\PresenceLight\

If one is there, delete it and try again.

**Comment by steveshallcross** [comment]
Thanks Isaac.

I did that, but I got the same result.

Browser window opened to log in, I enter my email address, and it says I
need to use a Work or School account.

If this is a problem no-one else is reporting, I might have a look at doing
a pull request myself and see if I can get the App to register with a
personal account in Azure.

Steve

**Reply by isaacrlevin** [reply, marked as answer]
Maybe I am not following. MSA accounts will not work currently, they have to be M365 accounts. More than happy to review a PR
